### PR TITLE
add exiting

### PR DESCRIPTION
**Pull request by lnkssr**
Added the ability to exit the application by clicking on the "x", the radio style has been saved.

### CODE REVIEW

File: radio.sh [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
+RED='\033[31m'
+RESET='\033[0m'
+
 echo "Keybinds:"
 echo "		p pause"
 echo "		9 add volume -2"
 echo "		/ add volume -2"
 echo "		0 add volume 2"
 echo "		* add volume 2"
+echo "                X exit for app"
```
Comment (KoNekoD) [inline radio.sh]: image

Please fix formating

File: radio.sh [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
-    printf "1. Anime[\033[31myggdrasilradio\033[0m]\n"
-    printf "2. Phonk[\033[31mbadradio\033[0m]\n"
-    printf "3. Phonk[\033[31mradiorecord.ru/station/phonk\033[0m]\n"
-	printf "4. Anime[\033[31manison.fm\033[0m]\n"
-	printf "5. Brazil Phonk[\033[31mUnknown\033[0m]\n"
-	printf "6. Lofi[\033[31m247onlineradio.com\033[0m]\n"
-	read -r -p "Choice: " answer
+    printf "1. Anime [${RED}YggdrasilRadio${RESET}]\n"
+    printf "2. Phonk [${RED}BadRadio${RESET}]\n"
+    printf "3. Phonk [${RED}RadioRecord.phonk${RESET}]\n"
```
Comment (KoNekoD) [inline radio.sh]: Its not single site, its single radio station `radiorecord.ru/station/phonk`

**Review by KoNekoD — APPROVED** [review]

**Comment by lnkssr** [comment]
All the flaws have been fixed

**Comment by KoNekoD** [comment]
@lnkssr Thank you, closing conversations was not necessary, I wanted to see your edits, and I think it's a sign of bad behavior

File: radio.sh [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
-    printf "1. Anime[\033[31myggdrasilradio\033[0m]\n"
-    printf "2. Phonk[\033[31mbadradio\033[0m]\n"
-    printf "3. Phonk[\033[31mradiorecord.ru/station/phonk\033[0m]\n"
-	printf "4. Anime[\033[31manison.fm\033[0m]\n"
-	printf "5. Brazil Phonk[\033[31mUnknown\033[0m]\n"
-	printf "6. Lofi[\033[31m247onlineradio.com\033[0m]\n"
-	read -r -p "Choice: " answer
+    printf "1. Anime [${RED}YggdrasilRadio${RESET}]\n"
+    printf "2. Phonk [${RED}BadRadio${RESET}]\n"
+    printf "3. Phonk [${RED}Radiorecord.ru/station/phonk${RESET}]\n"
```
Comment (KoNekoD) [inline radio.sh]:
```suggestion
    printf "3. Phonk [${RED}radiorecord.ru/station/phonk${RESET}]\n"
```

**Pull request merged**